template/src/components/dashboard/sites/DashboardSite.tsx

```diff
@@ -30,16 +56,59 @@ export const DashboardSite = observer(() => {
             >
                 {t("language.english")}
             </Button>
-            <div>
-                <Button
-                    onClick={() => {
-                        authStore.logout();
-                        pushRoute(BaseRoutes.ROOT);
-                    }}
-                >
-                    {t("common.logout")}
-                </Button>
-            </div>
-        </h1>
+        </div>
+    );
+
+    const loading = (
+        <>
+            <Button
+                variant="outlined"
+                onClick={async () => {
```

Comment:
in order to keep up with our best practices it would be nice to move the handlers into their own `handle`-function